doc/source/whatsnew/v0.24.2.rst

```diff
@@ -18,7 +18,7 @@ including other versions of pandas.
 .. _whatsnew_0242.regressions:
 
 Fixed Regressions
-^^^^^^^^^^^^^^^^^
```

Comment:
Looks like some git issues. Try reverting the changes to this file.

Reply:
Hmm...The master seems to reflect these changes in v0.24.2.rst as well. Am I missing something?